### PR TITLE
feat: add docker-compose.yaml and correct README.md

### LINKED ISSUE #1028: Change Artipie release workflow to include front service
As dashboard was moved to separate repository and runs separately now, we have to change Artipie release script and provide Artipie as service with dashboard included. There seems to be two approaches to achieve that:
- provide some docker-compose with Artipie and `front` service as docker images
- include `front` jar file into Artipie docker image

After preparing new release, if necessary, we have to change "Quickstart" section of the readme file accordingly and check other places: wiki, [CONTRIBUTING.md](https://github.com/artipie/artipie/blob/master/CONTRIBUTING.md) and other docs.

### PR DESCRIPTION
**Pull request by olenagerasimova**
Closes #1028 
Added docker-compose script to run both Artipie and Artipie front, corrected readme. 

Note, that I'm not a devops and have no idea about best practicies for such type of configurations.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -44,31 +44,35 @@ The following set of features makes Artipie unique among all others:
 # Quickstart
```
Comment (dgarus) [inline README.md:44]: Currently, we cannot host the data in Google Cloud and HuaweiCloud OBS...

**Review by dgarus — CHANGES_REQUESTED** [review]
@olenagerasimova, pay attention to my comment, please.

**Review by dgarus — DISMISSED** [review]
Thanks!
LGTM

**Comment by olenagerasimova** [comment]
@dgarus thanks, I've corrected part about storages. Have you tried to run docker-compose? Steps to check everything is working can be the following:
- run docker-compose
- go to dashboard: 
    * login with test user `artipie/artipie`
    * create new binary repository
- put some file into created repository (use `curl PUT` or smth similar)
- get the file from repository (use `curl GET` or smth similar)

**Comment by olenagerasimova** [comment]
@dgarus I've corrected docker-compose script, storages configuration and readme. 
As for [named volumes](https://docs.docker.com/storage/volumes/#start-a-service-with-volumes) 

> When you start a service and define a volume, each service container uses its own local volume. None of the containers can share this data if you use the local volume driver

Volumes also has some volumes drivers, but most of them are for AWS or Azure or some other cloud storage support, not for simple data sharing. 

That's why I simply added two volumes with some (linux consistent) local directory.

**Pull request merged**